modules/ROOT/pages/dataweave-types.adoc

```diff
@@ -469,6 +469,27 @@ The `Iterator` type is based on the https://docs.oracle.com/javase/8/docs/api/ja
 
 Note that like the Java class, the iterator is designed to be consumed only once. For example, if you then pass this value to a logger would result in consuming it and it would no longer be readable to further elements in the flow.
 
+[[dw_type_null]]
+== Null (dw::Core Type)
+
+DataWeave defines a `Null` data type for `null` values. It is important to note that `null` 
+is the value, while `Null` is the type. DataWeave functions do not recognize `Null` as
+a `null` value. 
```

Comment:
Just to reiterate once more/pound it into their heads...:
```suggestion
is the value, while `Null` is the type. DataWeave functions do not recognize the `Null` data type as
a `null` value. 
```

Reply:
A data type would never be the value. The point is that you can't pass "Null" and expect that to be treated as "null".